orianna.py

```diff
@@ -365,7 +365,6 @@ def get_role_emoji(summoner):
             emoji_name = duo_codes[summoner.role]
         else:
             emoji_name = player.lane.lower()
-
         emoji_code = emojis.get(emoji_name, 'Emoji not available')
```

Comment:
Adding `get()` removed KeyErrors showing up. Further testing would need to be done to confirm this.